packages/hash/README.md

```diff
@@ -147,6 +123,19 @@ yarn workspace @hashintel/block-name serve
 
 See https://blockprotocol.org/docs/developing-blocks
 
+## Development
+
+[//]: # (TODO: Pointers to where to update/modify code)
```

Comment:
Do we want to invest time now doing this? Basically we should give pointers to the various packages (for example, the frontend code is in packages/hash/frontend, and it _should_ have its own README with the relevant info so we stop duplicating it everywhere).

My guess is we want to wait for more of the repo restructuring work and @kachkaev and help with making this consistent.

Reply:
Agreed. Let's please capture this as an internal ticket and wait for the monorepo restructure work, as you suggest.

Reply:
I agree. We can add pointers once repo restructuring is done.

Reply:
I shall leave the TODO in.